### PR TITLE
Remove scoped style

### PR DESCRIPTION
**Pull request by adanilo**
Addresses issue #231

### CODE REVIEW

**Comment by adanilo** [comment]
Likely will want to change the example with 2 video elements and inline style due to the removal of 'scoped'. Suggested alternates most welcome.

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -10147,15 +10147,9 @@ zero or more <{track}> elements, then
...
-  &lt;style scoped&gt;
-  div { margin: 1em auto; position: relative; width: 400px; height: 300px; }
-  video { position; absolute; bottom: 0; right: 0; }
-  video:first-child { width: 100%; height: 100%; }
-  video:last-child { width: 30%; }
-  &lt;/style&gt;
-  &lt;div&gt;
-  &lt;video src="movie.vid#track=Video&amp;amp;track=English" autoplay controls mediagroup=movie&gt;&lt;/video&gt;
-  &lt;video src="movie.vid#track=sign" autoplay mediagroup=movie&gt;&lt;/video&gt;
+  &lt;div style="{ margin: 1em auto; position: relative; width: 400px; height: 300px; }"&gt;
```
Comment (arronei) [inline sections/semantics-embedded-content.include]: `style` attribute value should not contain `{ }`

**Comment by travisleithead** [comment]
Other than Arron's comment, this looks good. I did note one thing: in one example, you promote the styles to the head, but in the table example, you remove them entirely. Essential vs. non-essential styles?

**Comment by adanilo** [comment]
Good catch on the curlys, thanks. @travisleithead yes, the table example didn't need the styles at all but the other example breaks without them, and without scoped they need to be in the head.

**Pull request merged**